python/instrumentation/openinference-instrumentation-openai/src/openinference/instrumentation/openai/_attributes/_responses_api.py

```diff
@@ -166,6 +166,7 @@ def _get_attributes_from_response_computer_tool_call_param(
         yield f"{prefix}{MessageAttributes.MESSAGE_ROLE}", "tool"
         if (call_id := obj.get("call_id")) is not None:
             yield f"{prefix}{MessageAttributes.MESSAGE_TOOL_CALL_ID}", call_id
+        # TODO: should the role be "assistant" and args be obj.get("action")?
```

Comment:
not sure I follow these notes here.

Reply:
yeah this one didn't age well, will remove

```suggestion
```